pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -796,7 +797,7 @@ func TestNewFrameworkMultiPointExpansion(t *testing.T) {
 
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
-			fw, err := NewFramework(registry, &config.KubeSchedulerProfile{Plugins: tc.plugins})
+			fw, err := NewFramework(registry, &config.KubeSchedulerProfile{Plugins: tc.plugins}, wait.NeverStop)
```

Comment:
Why not use a stopCh and defer to close it, it looks like we're not going to recycle the goroutine with `wait.NeverStop`.

Reply:
This is not regressing existing tests (which aren't closing that now).
I want to fix the problem in production code and leave the cleanup for scheduler maintainers :)

Reply:
SG, I'll help to follow this.

Reply:
by production you also meant integration tests, right? Because they use `scheduler.New`

Reply:
Yes - I mostly meant "prod + non-unit tests"